ape_etherscan/explorer.py

```diff
@@ -55,21 +55,6 @@ def get_contract_type(self, address: AddressType) -> Optional[ContractType]:
 
         return contract_type
 
-    def get_account_transactions(self, address: AddressType) -> Iterator[ReceiptAPI]:
```

Comment:
Maybe the tests for this could be migrated to the query tests.

Reply:
I don't think this had tests